addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1780,6 +1780,22 @@ options.registry.ScrollButton = options.Class.extend({
     },
 });
 
+options.registry.ImageCaption = options.Class.extend({
+    defaultCaption: _t('Write a caption under your image'),
+    onFocus() {
+        this.$caption = this.$target.siblings('.o_we_image_caption');
+        if (!this.$caption.length) {
+            this.$caption = $(`<span class="o_we_image_caption d-block w-100 text-muted font-italic text-center pt-2">${this.defaultCaption}</span>`);
+            this.$caption.insertAfter(this.$target);
+        }
+    },
+    cleanForSave() {
+        if (this.$caption.text() === this.defaultCaption) {
+            this.$caption.remove();
+        }
+    }
+});
```

Comment:
Conventions and doc

Reply:
Feature has been dropped as discussed with FP